apsuite/orbcorr/calc_orbcorr_mat.py

```diff
@@ -87,20 +111,31 @@ def _get_respmat_line(
         quad[1, 2] = -ksxl/2
         quad[3, 0] = -ksyl/2
         half_cor = drift @ quad @ drift
+
+        # transfer matrix from 0 to the middle of corrector, Rc:
         rc_mat = half_cor @ rc_mat
 
+        # one tune matrix at the middle of corrector, Mc:
         mc_mat = np.linalg.solve(
             rc_mat.T, (rc_mat @ m_mat).T).T  # Mc = Rc M Rc^-1
+
+        # inverse: (1-Mc)^-1
         mci_mat = np.eye(mc_mat.shape[0], dtype=float) - mc_mat
 
         small = self.bpm_idx < corr
         large = np.logical_not(small)
 
+        # transfer matrix from corrector to BPM, Rc->b:
         rcbl_mat = np.linalg.solve(rc_mat.T, rb_mat.transpose((0, 2, 1)))
         rcbl_mat = rcbl_mat.transpose((0, 2, 1))
+
+        # if bpm is before corrector, Rc->b = Rb * M0 * Rc^-1
```

Comment:
it seems this comment does note reflect what is implemented. I think the comment is wrong, isn't it?

Reply:
I think it's not correct because I deleted one comment before that one. However, it reflects what is being calculated. What do you think about the following code?

```Python
# transfer matrix from corrector to BPM, Rc->b:
# first, calculate Rb * Rc^-1
rcbl_mat = np.linalg.solve(rc_mat.T, rb_mat.transpose((0, 2, 1)))
rcbl_mat = rcbl_mat.transpose((0, 2, 1))

# if bpm is before corrector, Rc->b = Rb * M0 * Rc^-1
rcbs_mat = rcbl_mat[small] @ mc_mat

# if bpm is after corrector, Rc->b = Rb * Rc^-1
rcbl_mat = rcbl_mat[large]
```

Reply:
We talked offline. @otavio-silveira agreed the comment was not clearly indicating what was implemented and will change it.